lint/file.go

```diff
@@ -127,7 +127,7 @@ type enableDisableConfig struct {
 }
 
 func (f *File) disabledIntervals(rules []Rule) disabledIntervalsMap {
-	re := regexp.MustCompile(`^\s*revive:(enable|disable)(?:-(line|next-line))?(:)?([^\s]*)?(\s|$)`)
+	re := regexp.MustCompile(`revive:(enable|disable)(?:-(line|next-line))?(?::([^\s]+))?`)
```

Comment:
This will match: `//random text revive:enable` as well? I'm not sure if we should enable this.

I suppose you had to do this because for:

```
//revive:disable
//revive:enable
```

The comment in the AST will be: `revive:disable revive:enable`?

Reply:
@chavacava 
>Other possible solution is to iterate over all Comments in the CommentGroup. At the Comment level the text is as in the source code (including //)

 This sounds cleaner to me. It'll still be a `O(n)` check but with a bit more manual work, I guess.

Reply:
@mgechev 
I've pushed the fix version that iterates over each comment in the CommentGroup.
This version only supports revive annotations on mono-line comments starting with `//`

I can extend it to cope with annotation on multi-line comments (`/* ... */`) but I need some definitions on what to accept as valid annotations in such comments.

```go
/*  revive:disable */

/* This is a comment
   revive:disable
   that expands over 3 lines */

/*
revive:disable
*/
```

Reply:
I think this is good ATM. Great job ⚡️